{"name": "Creating the new account", "status": "failed", "statusDetails": {"message": "AssertionError: Expected text: \"Account and Contact Information\", but got \"Email address already in use\".\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\create_login_03_07_2020.py\", line 120, in vrf_text_is_hr\n    context.app.create_login.vrf_text_is_hr(text)\n  File \"C:\\Everything\\My_docs\\Self_development\\IT\\Testing\\Automation_08_09_2019\\allivet_dt_03_05_2020\\pages\\create_login.py\", line 113, in vrf_text_is_hr\n    self.verify_text(text, *self.TEXT_FIELD)\n  File \"C:\\Everything\\My_docs\\Self_development\\IT\\Testing\\Automation_08_09_2019\\allivet_dt_03_05_2020\\pages\\base_page.py\", line 38, in verify_text\n    assert expected_text in actual_text, f'Expected text: \"{expected_text}\", but got \"{actual_text}\".'\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1584144510431, "stop": 1584144512424}, {"name": "Then Click MY ACCOUNT button", "status": "passed", "start": 1584144512424, "stop": 1584144512542}, {"name": "Then Click Sign In button", "status": "passed", "start": 1584144512542, "stop": 1584144515792}, {"name": "Then Click on Create an account tab", "status": "passed", "start": 1584144515792, "stop": 1584144517078}, {"name": "Then Input First Name FirstName", "status": "passed", "start": 1584144517079, "stop": 1584144517501}, {"name": "Then Input Last Name LastName", "status": "passed", "start": 1584144517501, "stop": 1584144517681}, {"name": "Then Input Telephone 4074354433", "status": "passed", "start": 1584144517682, "stop": 1584144517884}, {"name": "Then Input Mobile 4074354433", "status": "passed", "start": 1584144517884, "stop": 1584144518071}, {"name": "Then Input Account Name FirstName LastName", "status": "passed", "start": 1584144518071, "stop": 1584144518410}, {"name": "Then Input Email Address sanoy2006!d@rambler.ru", "status": "passed", "start": 1584144518410, "stop": 1584144518742}, {"name": "Then Input Password Password_Test", "status": "passed", "start": 1584144518742, "stop": 1584144518949}, {"name": "Then Confirm Password Password_Test", "status": "passed", "start": 1584144518949, "stop": 1584144519140}, {"name": "Then Input Address 6058 SW 18TH ST APT 4", "status": "passed", "start": 1584144519140, "stop": 1584144519508}, {"name": "Then Choose Country United States of America", "status": "passed", "start": 1584144519509, "stop": 1584144519532}, {"name": "Then Input Zipcode 33023", "status": "passed", "start": 1584144519533, "stop": 1584144519668}, {"name": "Then Choose State/Sity Fl, Hollywood", "status": "passed", "start": 1584144519668, "stop": 1584144519687}, {"name": "Then Click on Create Account button", "status": "passed", "start": 1584144519688, "stop": 1584144528045}, {"name": "Then Verify that text is here: Account and Contact Information", "status": "failed", "statusDetails": {"message": "AssertionError: Expected text: \"Account and Contact Information\", but got \"Email address already in use\".\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\create_login_03_07_2020.py\", line 120, in vrf_text_is_hr\n    context.app.create_login.vrf_text_is_hr(text)\n  File \"C:\\Everything\\My_docs\\Self_development\\IT\\Testing\\Automation_08_09_2019\\allivet_dt_03_05_2020\\pages\\create_login.py\", line 113, in vrf_text_is_hr\n    self.verify_text(text, *self.TEXT_FIELD)\n  File \"C:\\Everything\\My_docs\\Self_development\\IT\\Testing\\Automation_08_09_2019\\allivet_dt_03_05_2020\\pages\\base_page.py\", line 38, in verify_text\n    assert expected_text in actual_text, f'Expected text: \"{expected_text}\", but got \"{actual_text}\".'\n"}, "start": 1584144528046, "stop": 1584144528072}], "start": 1584144501262, "stop": 1584144530278, "uuid": "e6f03abb-0bcf-4db2-aeda-a71e51357cfa", "historyId": "5394734a3976682169dc14a7cf1e47d7", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Creation of a new account"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}